Week3/homework/step2-2.js

```diff
@@ -3,18 +3,31 @@
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
 
-  // Replace this comment and the next line with your code
+  for (let i = startIndex; i <= stopIndex; i++) {
+    numbers.push(i);
+  }
+  for (let i = 0; i < numbers.length; i++) {
+    if (numbers[i] % 3 === 0 && numbers[i % 5 === 0]) {
```

Comment:
This line causes the test for this strep to fail. Can you figure out what the error is?